libs/compute_cuda/include/hpx/compute/cuda.hpp

```diff
@@ -9,12 +9,12 @@
 #ifndef HPX_COMPUTE_CUDA_HPP
 #define HPX_COMPUTE_CUDA_HPP
 
+#include <hpx/cuda_support/get_targets.hpp>
+#include <hpx/cuda_support/target.hpp>
```

Comment:
This will cause the clang-format-check to complain. Please clang-format the module files.

Reply:
Ahh, I take this back - this file is not part of a module (yet), so things should be good.